dragonchain/transaction_processor/level_5_actions.py

```diff
@@ -107,6 +109,25 @@ def execute() -> None:
     # Only check confirmations if there are any pending
     check_confirmations()
 
+    # go through the failed claims backlog
+    _log.info("Trying to finalize any previously failed claims...")
+    process_claims_backlog()
+
+
+def process_claims_backlog() -> None:
+    claims = redis.hgetall_sync("mq:failed-claims", decode=False)
```

Comment:
hgetall? that's definitely not right, also why no decode?

Reply:
Should be using set.  Fixed.